fluent-validator/src/main/java/com/baidu/unbiz/fluentvalidator/GenericResult.java

```diff
@@ -23,6 +24,16 @@
      */
     protected List<T> errors;
 
+    public GenericResult() {
+        isSuccess = true;
+        errors = new ArrayList<T>();
```

Comment:
I think if you would like the errors to be an empty list not a NULL, maybe creating a new ResultCollector implementation can be a solution, so that it can fit your need. Although I still think it would be better with NULL errors list, and I admit and agree that NULL can be a bad thing, so I am thinking of introducing functional programming best practices, like Optional to solve the problem.